RumenTsenkov24_chess.py

```diff
@@ -0,0 +1,136 @@
+
+Домашно N2:
+Svetlin Mladenov
+•
+24.11
+100 точки
+Краен срок: Днес
+Create a data structure that represents a chess
+position. Use it to represent the starting position of a chess game.
+Print the position to screen. A position in chess is a board with a
+particular configuration of the pieces on it.
+
+Notes:
+You may use termcolor to print colors.
+When submitting the PR please add a screenshot of the position printed.
+You can also use emojis to print the figures on the board.
+
+termcolor · PyPI
+https://pypi.org/project/termcolor/
+Коментари за курса
+Вашата работа
+Предадени
+
+chess.py
+Текст
+
+Chess_starting_possitions.jpg
+Изображение
+Частни коментари
+from string import ascii_uppercase
```

Comment:
What's all that :arrow_up: ? Why have you included it here? This file is not a valid python file, it doesn't run.

Also you have committed it to the wrong place. I moved it to
`software-engineering-2223/homework/12_V/24_Rumen_Tsenkov`.
Commit future homework in that directory.